src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ExternalMethodologyViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Admin.Requests;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+
+public record ExternalMethodologyViewModel
+{
+    public string Title { get; init; } = string.Empty;
+
+    public string Url { get; init; } = string.Empty;
+
+    public ExternalMethodologyViewModel()
+    {
+    }
+
+    public ExternalMethodologyViewModel(ExternalMethodology externalMethodology)
+    {
+        Title = externalMethodology.Title;
+        Url = externalMethodology.Url;
+    }
+
+    public ExternalMethodologyViewModel(ExternalMethodologySaveRequest externalMethodology)
+    {
+        Title = externalMethodology.Title;
+        Url = externalMethodology.Url;
+    }
```

Comment:
I don't think it really matters but `return new ExternalMethodologyViewModel(publication.ExternalMethodology)` after saving the updated publication would mean this isn't needed.